### ISSUE TITLE
Complicated task flow between routing requests task and fault injection task

### ISSUE THREAD

**Issue by ijsnellf** [istio/istio.github.io#241]
There are cyclical references in the routing requests and fault injection tasks. Following the steps in the request routing task directs you to the fault injection task, which in turn directs you back to the routing routing task. Additionally, the next task after the routing task is the fault injection task. This flow is hard to follow. We should try to separate the tasks more cleanly.

https://istio.io/docs/tasks/request-routing.html
https://istio.io/docs/tasks/fault-injection.html

**Comment by rshriram** [comment]
Assigning to @ijsnellf since he spotted the issue. ccing @frankbu just for reference.

**Comment by sakshigoel12** [comment]
@ijsnellf can you add an ETA?

**Comment by rshriram** [comment]
This has been fixed..
